File: Frontend/Angular/src/app/pages/khoa-hoc/model/khoa-hoc.model.ts
import { BaseModel } from "src/app/_metronic/core/models/_base.model";

export class KhoaHocModel extends BaseModel {
    id: number;
    TenKhoaHoc: string;
    NamHoc: number | string;
    CachViet: string;
    Disable: boolean;
    CreatedBy: string;
    CreatedDate: Date | string;
    TenNamHoc: string;

    clear() {
        this.id = 0;
        this.TenKhoaHoc = '';
        this.NamHoc = 0;
        this.CachViet = "";
        this.Disable = true;
        this.CreatedBy = "";
        this.CreatedDate = new Date();
        this.TenNamHoc = "";
    }
}

File: SampleCodeAPI/Model/KhoaHocModel.cs
using System.Text.Json.Serialization;

namespace SampleCodeAPI.Model
{
    public class KhoaHocModel
    {
        public int? id { get; set; }
        public String TenKhoaHoc { get; set; }
        public String? TenNamHoc { get; set; }
        public int NamHoc { get; set; }
        public string? CachViet { get; set; }
        public bool? Disable { get; set; }
        public string? CreatedBy { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string? UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public string? DeletedBy { get; set; }
        public DateTime? DeletedDate { get; set; }
    }
}
